constantinCurtis/src/main/java/inputOutputTask/CountMethod.java

```diff
@@ -0,0 +1,35 @@
+package inputOutputTask;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class CountMethod {
+    public static int countMethod(String word, File file) throws IOException {
+        int result = 0;
+        BufferedReader bufRead = new BufferedReader(new FileReader(file));
+        String str;
+        while ((str = bufRead.readLine()) != null) {
+            if (str.equalsIgnoreCase(word)) {
+                result++;
+            }
+        }
+        return result;
+    }
+
+    public static int countNio(String word, Path path) throws IOException {
```

Comment:
I don't think it's a `countNio`
using `Files.lines` will be more like

Reply:
Thank you, done.